packages/tour-kit/package-lock.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
Make sure that you're using `yarn install` to update dependencies! We shouldn't have any `package-lock.json` files in `wp-calypso` -- everything will be automatically added to the root `yarn.lock` file :)